src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -88,13 +147,35 @@ public function getItem(string $resourceClass, $id, string $operationName = null
             ++$i;
         }
 
-        if (!$fetchData || $manager instanceof EntityManagerInterface) {
-            return $manager->getReference($resourceClass, $identifiers);
+        return $identifiers;
+    }
+
+    /**
+     * @param ObjectManager $resourceManager
+     * @param string        $resourceClass
+     *
+     * @return \Doctrine\ORM\QueryBuilder
+     */
+    private function retrieveQueryBuilder(ObjectManager $resourceManager, string $resourceClass)
+    {
+        $repository = $resourceManager->getRepository($resourceClass);
+
+        if ($repository instanceof  EntityRepository) {
```

Comment:
@dunglas should I check against `EntityRepository` or try if it has the `createQueryBuilder()` method?